packages/mobile/src/alert/actions.ts

```diff
@@ -60,6 +61,16 @@ export const showErrorInline = (error: ErrorMessages, i18nOptions?: TOptions): S
   underlyingError: error,
 })
 
+// Useful for showing a more specific error if its a documented one, with
+// a fallback to something more generic
+export function* showErrorMessageOrFallback(error: any, fallback: ErrorMessages) {
```

Comment:
Nice, we use this pattern pretty often! Does this need to be a generator?

Reply:
It does so it can yield put to `showError`.
An alternative would be have it just return the ErrorMessage, which we can yield put, but seemed nice to have it all together :)

Reply:
Oh I see I assumed the latter.

IMO it's a little non-intuitive that we would `yield put(showError)` and `yield call(showErrorMessageOrFallback)` but that's more of a nit so up to you :)

Reply:
Cool I'll change this then, it'd be good for it to match people's first intuition :)

Reply:
`yield put(showErrorMessageOrFallback)` would be more intuitive for me as well